helix-view/src/document.rs

```diff
@@ -1411,6 +1465,16 @@ impl Document {
             apply_inlay_hint_changes(padding_after_inlay_hints);
         }
 
+        for text_annotation in self.color_swatches.values_mut() {
```

Comment:
Do we need to get mutable values here?

Reply:
yes, since `apply_inlay_hint_changes` takes a `&mut`